src/hagl.c

```diff
@@ -48,7 +49,7 @@ SPDX-License-Identifier: MIT
 #include "window.h"
 #include "hagl.h"
 #include "hagl_hal.h"
-
+static const char * TAG="hagl";
```

Comment:
Same here. This is ESP logger specific code. HAGL is hardware / platform agnostic.